sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -0,0 +1,950 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup net_ng_ndp
+ * @{
+ *
+ * @file
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+#include <string.h>
+
+#include "byteorder.h"
+#include "net/ng_icmpv6.h"
+#include "net/ng_ipv6.h"
+#include "net/ng_netbase.h"
+#include "net/ng_netif/hdr.h"
+#include "net/ng_pktbuf.h"
```

Comment:
"net/ng_netif/hdr.h" and "net/ng_pktbuf.h" are already included in "net/ng_netbase.h".